applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -606,6 +606,21 @@ The current maximum number of Bluetooth LE connections that can be selected usin
 
                west build -b nrf7002dk/nrf5340/cpuapp -- -DCONFIG_BRIDGED_DEVICE_BT=y -DEXTRA_CONF_FILE="overlay-bt_max_connections_app.conf" -Dipc_radio_EXTRA_CONF_FILE="overlay-bt_max_connections_net.conf"
 
+Configuring Bluetooth LE connection and scan parameters
+-------------------------------------------------------
+
+You can set your own Bluetooth connection parameters instead of accepting the default ones requested by the peripheral device. To do this, enable the ``BRIDGE_FORCE_BT_CONNECTION_PARAMS=y`` option, which allows you to modify the parameters.
```

Comment:
```suggestion
You can set your own Bluetooth LE connection parameters instead of accepting the default ones requested by the peripheral device. To do this, enable the ``BRIDGE_FORCE_BT_CONNECTION_PARAMS=y`` option, which allows you to modify the parameters.
```